### PR TITLE
fix responsive image from team page. from issue #623

### PR DESCRIPTION
**Pull request by adiwahyudi02**
Fix responsive image from team page.
Solved #623

### CODE REVIEW

**Comment by adiwahyudi02** [comment]
result

**Review by pranav1698 — COMMENTED** [review]
Please write a description to your pr and link an issue which you have solved.

**Comment by adiwahyudi02** [comment]
Could you please check it now? @pranav1698 
https://adiwahyudi02.github.io/fossasia.org/team/

File: css/theme-lava.css [hunk 1]
```diff
@@ -1341,9 +1341,19 @@ nav .container {
...
+} */
+/* add */
 .speaker .image-holder {
+  width: 300px;
+  height: 300px;
+  margin: auto;
   margin-bottom: 12px;
 }
+.speaker span{ 
+  text-align: center;
```
Comment (pranav1698) [inline css/theme-lava.css]: Same here

File: css/theme-lava.css [hunk 2]
```diff
@@ -1341,9 +1341,19 @@ nav .container {
   -webkit-transition: all 0.3s ease-out;
   -moz-transition: all 0.3s ease-out;
 }
+/* .speaker .image-holder {
```
Comment (pranav1698) [inline css/theme-lava.css]: No need to add another block of code 
You can insert your changes in the same block

**Review by pranav1698 — CHANGES_REQUESTED** [review]

**Comment by adiwahyudi02** [comment]
I have updated it. please review again @pranav1698

**Review by pranav1698 — APPROVED** [review]

**Comment by pranav1698** [comment]
Please wait for someone with write access to merge the pr
@mariobehling

**Pull request merged**